model/conda.yml

```diff
@@ -0,0 +1,21 @@
+name: spotlight
+channels:
+  - conda-forge
+  - https://nexus-uclhtredev.uksouth.cloudapp.azure.com/repository/conda-repo/main/
+  - https://nexus-uclhtredev.uksouth.cloudapp.azure.com/repository/conda-mirror/main/
+dependencies:
+  - python=3.8
+  - numpy
+  - pip
+  - scikit-learn
+  - scipy
+  - pandas
+  - pip:
```

Comment:
Could please remove the `polars` (the template doesn't require `polars`) and the duplicate `mlflow` requirements here